src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,12 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage {
+    public final By LOGOUT = By.id("PH_logoutLink");
+
+    public void logout(WebDriver driver){
+        driver.findElement(LOGOUT).click();
```

Comment:
Create constructor, which will take WebDriver driver as parameter and save to class field. This WebDriver field will be used in all methods of class. It will help to avoid passing WebDriver to each method of the class